staff/aiida-montero/Challenges/calculator-app-grid/style.css

```diff
@@ -0,0 +1,84 @@
+*{margin: 40px
+    padding:0px¨
+    }
+
+
+h1 {
+
+    border: 1.5px solid aquamarine;
+    padding-block: 50px;
+    background: rosybrown;
+    text-align:center;
+    color: white;
+    border-radius: 10px;
+    font-family: apple-system;
+    letter-spacing: 2px;
+    font-style: italic;
+  
+    }
+
+    .contenedor{
+ 
+        text-align: center;
+        max-width: 1000px;
+        border: 3px solid aqua;
+        margin: auto;
+        line-height: 50px;
+        background: grey;
+        
+
+
+    }
+    .contenedor > button {
+
+        display: block;
+        padding: 0;
+        color: :black;
+        background:grey;
+        box-shadow:2px 2px 10px black;
+        height: 40px;
+      
+       
```

Comment:
remove empty lines to make it cleaner and readable.

Reply:
Listo Beto :)